client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -8,6 +8,7 @@
 		padding: 0;
 		display: flex;
 		flex-direction: column;
+		max-width: none;
```

Comment:
No problem, let's go ahead. My guess is that the `max-height: 90%` is a bug introduced by #15718 rather than some intentional design change.

The default styles should be for mobile and the `max-height` and `max-width` should be added only for larger screen sizes with `@media` breakpoints.